include/tesla.hpp

```diff
@@ -1028,7 +1028,9 @@ namespace tsl {
                     ASSERT_FATAL(viSetLayerPosition(&this->m_layer, cfg::LayerPosX, cfg::LayerPosY));
                     ASSERT_FATAL(nwindowCreateFromLayer(&this->m_window, &this->m_layer));
                     ASSERT_FATAL(framebufferCreate(&this->m_framebuffer, &this->m_window, cfg::FramebufferWidth, cfg::FramebufferHeight, PIXEL_FORMAT_RGBA_4444, 2));
+                    ASSERT_FATAL(setInitialize());
                     ASSERT_FATAL(this->initFonts());
+                    ASSERT_FATAL(setExit());
```

Comment:
`setExit()` returns `void`. This most likely doesn't compile and if it does somehow, ASSERT_FATAL wouldn't do what it's supposed to.

Reply:
resolved